documentation/src/main/asciidoc/user_guide/equivalence.adoc

```diff
@@ -1,4 +1,4 @@
-==  Storing objects (e.g. arrays) with custom Equivalence functions
+==  Object Equivalence
```

Comment:
I want titles to be concise. I think "Customizing key/value comparisons" is better.